apps/backend/src/libs/modules/encryption/encryption.service.ts

```diff
@@ -0,0 +1,55 @@
+import bcrypt from "bcrypt";
+
+import { ApplicationError } from "~/libs/exceptions/exceptions.js";
+
+import { EncryptionServiceMessage } from "./libs/enums/enums.js";
+
+const DEFAULT_SALT_ROUNDS = 10;
+
+class EncryptionService {
+	private saltRounds: number;
+
+	public constructor() {
```

Comment:
We should pass salt rounds as parameters to constructor. Also we don't use process.env directly, we have a config module for this.